### PR TITLE
do not throw error for flow editor view when no org in context

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/flows/views.py [hunk 1]
```diff
@@ -884,7 +884,7 @@ def get_context_data(self, *args, **kwargs):
                 context['mutable'] = True
 
             org = self.request.user.get_org()
-            context['has_airtime_service'] = bool(org.is_connected_to_transferto())
```
Comment (nicpottier) [inline temba/flows/views.py]: Seems we could just derive this from the org on the flow no?

File: temba/flows/views.py [hunk 2]
```diff
@@ -884,7 +884,10 @@ def get_context_data(self, *args, **kwargs):
                 context['mutable'] = True
 
             org = self.request.user.get_org()
-            context['has_airtime_service'] = bool(org.is_connected_to_transferto()) if org else False
+            if not org:
+                org = self.object.org
+
+            context['has_airtime_service'] = bool(org.is_connected_to_transferto())
```
Comment (nicpottier) [inline temba/flows/views.py]: Should we just always use the flow org? I mean that's going to be the most correct isn't it?

Comment (norkans7) [inline temba/flows/views.py]: Ok

**Comment by nicpottier** [comment]
:neckbeard:

**Pull request merged**